composer.json

```diff
@@ -1,25 +1,41 @@
 {
-    "name": "mollie/mollie-woocommerce",
+    "name": "mollie/payments-for-woocommerce",
     "type": "wordpress-plugin",
-    "license": "GLP-v2",
+    "license": "GPL-2.0-or-later",
+    "authors": [
+        {
+            "name": "Mollie B.V.",
+            "email": "info@mollie.com"
+        },
+        {
+            "name": "Inpsyde Gmbh",
+            "email": "hello@inpsyde.com"
+        }
+    ],
     "minimum-stability": "dev",
     "require": {
-        "php": ">=5.6"
+        "php": "^5.6.40 | ^7.0",
+        "mollie/mollie-api-php": "^2.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^5",
-        "brain/monkey": "^2.0@dev",
-        "ptrofimov/xpmock": "^1"
```

Comment:
Why those packages are not included? We need them for testing

"phpunit/phpunit": "^5",
"brain/monkey": "^2.0@dev",
"ptrofimov/xpmock": "^1"

Reply:
I could not have removed these, because when I started working on this task, there was no Composer configuration. Most likely, the later changes weren't merged correctly onto this branch.